yew/src/agent/mod.rs

```diff
@@ -93,6 +93,12 @@ pub trait Bridge<AGN: Agent> {
     fn send(&mut self, msg: AGN::Input);
 }
 
+/// Type alias for the boxed [`Bridge`] returned by [`Agent::bridge`][`bridge`]
+///
+/// [`Bridge`]: ./trait.Bridge.html
+/// [`bridge`]: ./trait.Bridged.html#tymethod.bridge
+pub type DynBridge<AGN> = Box<dyn Bridge<AGN>>;
```

Comment:
Would this be clearer?
```suggestion
pub type AgentBridge<AGN> = Box<dyn Bridge<AGN>>;
```

Reply:
Probably. I didn't want to consume the name in case a non-dynamic solution comes along.